tests/formats/cif.cpp

```diff
@@ -0,0 +1,107 @@
+// Chemfiles, a modern library for chemistry file reading and writing
+// Copyright (C) Guillaume Fraux and contributors -- BSD license
+
```

Comment:
this should be wrapped in `#include <chemfiles.hpp> #idndef CHFL_DISABLE_GEMMI ... #endif`